pkg/controller/datavolume/util.go

```diff
@@ -151,6 +160,37 @@ func copyStorageAsPvc(log logr.Logger, storage *cdiv1.StorageSpec) *v1.Persisten
 	return pvcSpec
 }
 
+// Gets either the bound PV or an available satisfying PV
+func getPV(c client.Client, pvcSpec *v1.PersistentVolumeClaimSpec, pvcName, pvcNamespace string) (*corev1.PersistentVolume, error) {
+	if pvcSpec.StorageClassName == nil {
+		return nil, fmt.Errorf("PVC spec is missing StorageClassName")
```

Comment:
Is this really an error?  Can't storageclass name be nil for either `spec.pvc` or `spec.storage`?

Reply:
Is that tested?

Reply:
It's not an error, should return nil here. Fixed.